homeassistant/components/deconz/light.py

```diff
@@ -227,6 +227,16 @@ def __init__(self, device, gateway):
 
         self._unique_id = f"{group_id_base}-{self._device.deconz_id}"
 
+    @property
+    def max_mireds(self):
+        """Group don't report ctmax."""
+        return super(DeconzLight, self).max_mireds
```

Comment:
```py
return super().max_mireds
```

Reply:
No that would return the max_mireds of DeconzLight which is what is breaking

Reply:
Ok. We should consider splitting up the inheritance tree so that the deconz group light doesn't inherit properties from a parent that the group light doesn't want to inherit from. Ie add a common deconz light class that both the deoncz light and the deconz group light can inherit from.

This can be a separate PR.

Reply:
Sure, I can fix that

Reply:
I redid the fix just to make it easier to not forget how it works :)